.hintrc

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
What is this file? Is it really necessary?

Reply:
It's the configuation file of JSHint, use to specify the code style, It should start using until after a few commits later, I will delete it in this pr.

Reply:
Fixed.